webrtc.html

```diff
@@ -1862,6 +1880,52 @@
         </dl>
       </section>
     </section>
+    <section>
+      <h3>Session Negotiation Model</h3>
+      <p>Many changes to state of an <code><a>RTCPeerConnection</a></code> will
+        require communication with
+        the remote side via the signaling channel, in order to have the desired
+        effect. The app can be kept informed as to when it needs to do signaling,
+        by listening to the negotiationneeded event.</p>
+      <section>
+        <h4>Setting Negotiation-Needed</h4>
+        <p>If an operation is performed on an <code><a>RTCPeerConnection</a></code>
+          that requires signaling,
+          the connection will be marked as needing negotiation. Examples of such
+          operations include adding or stopping a track, or adding the first
+          data channel.</p>
```

Comment:
I'll argue once more: I think we should not limit Negotiation-Needed to happen _only_ as a result of an operation performed on the PC. If the UA for some reason needs the session to be renegotiated, it should be allowed to fire this event.

I propose changing
"If an operation is performed on an RTCPeerConnection that requires signaling, the connection will be marked as needing negotiation. Examples of such operations include adding or stopping a track, or adding the first data channel."
to
"Whenever an RTCPeerConnection requires signaling, the connection will be marked as needing negotiation. This usually happens as a result of performing an operation such adding or stopping a track, or adding the first data channel on the RTCPeerConnection, but can also happen as a result of internal changes in the implementation (such as resources becoming unavailable in runtime)."